src/main/webapp/dataset.xhtml

```diff
@@ -23,7 +23,8 @@
             <ui:param name="showMessagePanel" value="#{true}"/>
             <ui:param name="showPublishLink" value="#{DatasetPage.workingVersion == DatasetPage.dataset.latestVersion 
                                                       and DatasetPage.dataset.latestVersion.versionState=='DRAFT'
-                                                      and permissionsWrapper.canIssuePublishDatasetCommand(DatasetPage.dataset)}"/>
+                                                      and permissionsWrapper.canIssuePublishDatasetCommand(DatasetPage.dataset)
+                                                      and DatasetPage.valid}"/>
```

Comment:
@ErykKul  Sorry about the confusion, but when I requested "graying out" the publish button, I was really asking for it to be disabled. We want to show it but the logic for disabling it is around 319 and a bit beyond.